### ISSUE TITLE
Can I Remove Romless Games?

### ISSUE THREAD

**Issue by ShatteredDisk** [yoshisuga/MAME4iOS#408]
Hello!

I am currently trying to make a full MAME4iOS ROM collection. All of the ROMs I collect should at least partially work. I have found that, for me, the ROM-less games do not run correctly or have major problems. I would like to hide them or delete them. I have tried editing and deleting the “romless.ini” file, but every time I start the app it comes back. (I assume that’s something in the app’s source code.)

TL;DR: Is there any way to hide or remove ROM-less games permanently?

Thanks,
~ ShatteredDisk

**Comment by ToddLa** [comment]
TL;DR - No romless machines will always show up. 

There are two types of romless machines, arcade and console.

NOTE we need these romless machines to be able to play game carts (ie NES, 2600, ....)

You can hide consoles by selecting `Hide BIOS` in Settings.

As for the three arcade machines (Breakout and Pong) what does not work with them?

It should be possible to edit `romless.ini` just make sure the first line is the MAME version and it won't be re-generated. 

It would not be hard to hide these from view if really needed.

**Comment by ShatteredDisk** [comment]
Update: Hide BIOS was already enabled

**Comment by ToddLa** [comment]
Hide BIOS will not remove Breakout and Pong

If you edit romless.ini and remove just those three lines. That should do what you want
